ovs/lib/albacontroller.py

```diff
@@ -251,7 +251,10 @@ def add_cluster(alba_backend_guid):
 
         etcd_key = AlbaController.ETCD_NR_OF_AGENTS_KEY.format(alba_backend_guid)
         nr_of_storage_nodes = len(AlbaNodeList.get_albanodes())
-        EtcdConfiguration.set(etcd_key, nr_of_storage_nodes)
```

Comment:
We should not manually set the amount of storage nodes to 1 if none are present
The core problem here has most likely something to do with the order in which our hooks are executed during ovs setup

OVS setup used to model the ASD nodes right away, but since the introduction of the ASDmanager setup, this isn't done right away anymore. Something to investigate and revert this change